G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/ExcelImportText/ShouldImportTextIntoExcel_WhenCSVFileIsProvided.G1ANT

```diff
@@ -0,0 +1,16 @@
+﻿excel.open
+
+♥filePath = ‴♥environment⟦TEMP⟧\testcsv.csv‴
+
+text.write text ‴"1","2","3"‴ filename ♥filePath
+
+excel.importtext path ♥filePath destination ‴A1‴ delimiter ‴,‴
+
+excel.getrow row 1 result ♥usedCells
+
+for counter ♥a from 1 to 3 step 1
+    test ⊂♥usedCells⟦System.Convert.ToChar(96+♥a).ToString()⟧ == ♥a⊃
```

Comment:
`.ToChar('a' + ♥a - 1)` should also work and you will omit the magic value 96.
BTW won't the `♥a.ToString()` do exactly the same?

Reply:
If the test will fail you will leave both the file and open instance of excel. Please handle the error in some other way, so both the `file.delete` and `excel.close` will be called.
This remark applies to all other places where `test` is called before performing the test cleanup.